src/pages/Feedback/Feedback.js

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import {
+  Grommet,
+  Anchor,
+  Box,
+  Paragraph,
+  Grid,
+  Card,
+  Heading,
+  Image,
+  Main,
+} from 'grommet';
+import {
+  ClearOption,
+  Configure,
+  Camera,
+  CloudUpload,
+  Selection,
+  BarChart,
+  Github,
+  Slack,
+} from 'grommet-icons';
+import { grommetToolsTheme } from '../../theme.js';
+import AppHeader from '../../components/AppHeader.js';
+
+const Feedback = (props) => (
+  <Grommet background="gradient" className="App" theme={grommetToolsTheme}>
```

Comment:
The Application container should use the Grommet component, and there is no need to use it again inside the app.

Reply:
Removed Grommet component from Feedback.js and moved it to App.js

Reply:
Did you verify there should be one grommet component across the app?

Reply:
Yes there is only one